Technical review: Document crash reporting api (#43791)
mdn/content: files/en-us/web/api/window/crashreport/index.md

---
title: "Window: crashReport property"
short-title: crashReport
slug: Web/API/Window/crashReport
page-type: web-api-instance-property
browser-compat: api.Window.crashReport
---

{{securecontext_header}}{{APIRef("Reporting API")}}

> The **`crashReport`** read-only property of the {{domxref("Window")}} interface returns a {{domxref("CrashReportContext")}} object that enables arbitrary data to be recorded for the current top-level browsing context, which is then added to a {{domxref("CrashReport")}} and sent to a reporting endpoint when a browser crash occurs.

Reviewer: Consider making this into shorter sentences. 

```suggestion
The **`crashReport`** read-only property of the {{domxref("Window")}} interface returns a {{domxref("CrashReportContext")}} object that enables arbitrary data to be recorded for the current top-level browsing context.
The data is then included in {{domxref("CrashReport")}} objects that are sent to a reporting endpoint when a browser crash occurs.
```
